internal/snmp/testdata/mibs/SNMPv2.mib

```diff
@@ -0,0 +1,864 @@
+-- *****************************************************************
+-- SNMPv2-MIB.my:  Management Information Base (MIB) for the
+--                 Simple Network Management Protocol (SNMP)
+--
+-- March 2006,  Wen Xu
+--
+-- Copyright (c) 2006 by cisco Systems, Inc.
+-- All rights reserved.
+-- *****************************************************************
+--
+-- This mib was extracted from RFC 3418
+
+
```

Comment:
We can't accept files into the project that are copyright someone else.

I understand MIB files published as RFCs (like this) are copyright the Internet Society. They allow the MIB files to be extracted and used under certain legal conditions. To meet the conditions Linux distributions often don't include the MIB files. Instead, they have tools that download the RFCs and extract the MIBs (see debian package snmp-mibs-downloader). If we wanted to use MIBs from RFCs in our unit tests, I think we may need to do something similar.

We don't need the real MIBs for these tests. If we don't use the real MIBs we avoid copyright questions. That's why we wrote custom simple test mibs to use in unit tests (test2 and test.mib in plugins/inputs/snmp_trap).

Reply:
Indeed, I agree!